language/stdlib/modules/LibraAccount.move

```diff
@@ -218,6 +218,8 @@ module LibraAccount {
             // travel rule does not apply for intra-VASP transactions
             VASP::parent_address(sender) != VASP::parent_address(payee)
         ) {
+            // If either of the VASPs are expired, then abort
+            assert(!VASP::is_expired(sender) && !VASP::is_expired(payee), 9000); // TODO: proper error code
```

Comment:
Should we also prohibit transfers under the dual attestation threshold (i.e., move this before the `if`)?

Reply:
Mmm I was going back and forth on it. My thinking was that by having it under the if, the "fail state" of a VASP is a UHW. I'll go for the more conservative approach at this point (move it above), and then if we want to relax it later it should be easier.